### ISSUE TITLE
Incomplete source code for partner

### ISSUE THREAD

**Issue by missing233** [perilouswithadollarsign/cstrike15_src#8]
These source code is provided to partners.
Valve removed the econ system from this source code. Compiled files can't be used for played normally.
Similarly, compared with src2007, these source code has removed many parts.

**Comment by kaoti** [comment]
btw, you know ehere i need to put the depots files ?

**Comment by missing233** [comment]
In game/shared/econ/econ_game_account_clienet.cpp
You can find 
`/** Removed for partner depot **/`


F*ck Valve :D

**Comment by kaoti** [comment]
Lol, fuck me sideways twice

**Comment by missing233** [comment]
> btw, you know ehere i need to put the depots files ?

Anywhere.
 Then execute createsolution.bat

**Comment by kaoti** [comment]
Anywhere in the disk or in the main folder xD

**Comment by missing233** [comment]
> Anywhere in the disk or in the main folder xD

Yep
just clone the code and execute createsolution.bat
Don't forget to compile libcef and cryptopp

**Comment by kaoti** [comment]
Ok, thx

**Comment by missing233** [comment]
> Anywhere in the disk or in the main folder xD

then copy the game folder to the game root(2017 Ver.)

**Comment by HowToDoThis** [comment]
Just make sure, offline testing about everything, else VAC waiting you~

**Comment by kaoti** [comment]
i already stoped steam processes

**Comment by HowToDoThis** [comment]
and don't join any about VAC server related, auto blocked by client or verification by VAC

**Comment by perilouswithadollarsign** [comment]
a quote from another issue: "this is not up to date source, it can be updated to a certain extent but csgo is updated quite frequently so it wont ever be fully up to date unless another leak happens, which i highly doubt will ever come". do not expect this to be fully working. this isnt uploaded for people to paste the game everywhere, its to learn from. also yes, it should be fairly obvious that this will not work for playing multiplayer lol

**Comment by blueclouds8666** [comment]
do you guys think that modifying the code in order to remove the econ system dependency is possible? because otherwise it's going to fail on the compiling process.

**Comment by Titaniumtown** [comment]
could someone make a torrent for the files we need from the depot?

**Comment by Titaniumtown** [comment]
It would be super useful

**Comment by bxff** [comment]
@blueclouds8666 yes, we are still kinda of in the stone age for it...
